standard/docs/en/extensions/index.md

```diff
@@ -14,66 +14,23 @@ All core and community extensions are listed in the [extensions registry](https:
 
 ## Using extensions
 
-Extensions are applied by adding their URLs to the `extensions` array in the release or record package. To use one or more extensions, select them from the extension lists below and include the following in your release or record package metadata:
-
-<style><!--
-#using-extensions div[class^='highlight'] pre {
-   white-space: pre-wrap;  
-   white-space: -moz-pre-wrap;  /* Mozilla, since 1999 */
-   white-space: -pre-wrap;      /* Opera 4-6 */
-   white-space: -o-pre-wrap;    /* Opera 7 */
-   word-wrap: break-word;       /* Internet Explorer 5.5+ */
-}
---></style> 
-
-```json
-{
-    "extensions":[],
-    "releases":[]
-}
-```
+Extensions are applied by adding their URLs to the `extensions` array in the release package or record package. You can discover extensions and read their documentation using the [Extension Explorer](https://extensions.open-contracting.org/en/).
 
-## Core extensions
+This version of OCDS uses these specific versions of core extensions:
 
 ```eval_rst
-.. extensionselectortable::
-   :group: core
-```
-
-## Community extensions
+ .. extensionexplorerlinklist::
 
-```eval_rst
-.. extensionselectortable::
-   :group: community
 ```
 
+## Developing your own extensions
 
-## More information
+If you have additional fields to include in your data which are not covered by an existing extension, you may create a new extension.
```

Comment:
I suggest making the meaning of 'covered by' explicit, making it explicit that additional fields should be included in OCDS data and changing the recommendation to create a new extension from 'may' to 'should'. 

> If you have additional fields which cannot be mapped to the OCDS schema or an existing extension, you should include these in your OCDS data and create a new extension to document their structure and meaning.

Reply:
Done in f3cd65c